src/stores/SearchStore.ts

```diff
@@ -45,7 +45,7 @@ export type SearchResult = EventTreeNode | EventMessage;
 
 export type SearchHistory = {
 	timestamp: number;
-	results: Array<SearchResult>;
+	results: Record<string, Array<SearchResult>>;
```

Comment:
Some users already have search results saved in localstorage and their will not be able to use app unless they clear local storage themselves

i got
Uncaught TypeError: Cannot read property 'messageId' of undefined SearchPanelResults.tsx:64

Reply:
We'll think about it. @YourWeebyNeighbor says we can validate localStorage data and clear it if we got an invalid data format. It'll be resolved in the future